automatilib/cola_auth/i_dot_ai_cola_backend.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import ModelBackend
+
+LOGGER = logging.getLogger(__name__)
+
+User = get_user_model()
+
+
+class AuthenticationBackend(ModelBackend):
+    def authenticate(self, request, username=None, password=None, **kwargs):
+        """
+        Get user response given by JWT token from kwargs and update the user based on this
+        :param request: The HTTP request that is requesting authentication
+        :param username: The users username (not used)
+        :param password: The users password (not used)
```

Comment:
Is there a reason we're specifying username and password if they're not being used?

Reply:
They're part of the required signature of the function. I can remove it if we're happy with squiggly lines in the IDE